### PR TITLE
docs: fixed typos on Noto Serif

### PR DESCRIPTION
**Pull request by orion-cengage**

### CODE REVIEW

**Review by joasmi723 — APPROVED** [review]

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 1]
```diff
@@ -515,7 +515,7 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
       <p>
         <strong>Body / Large</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:518]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 2]
```diff
@@ -535,7 +535,7 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
       <p>
         <strong>Body / Medium</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:538]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 3]
```diff
@@ -545,15 +545,15 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
+        This size is great for creating contrast between the main paragraph of
+        the passage and secondary information.
       </Paragraph>
     </div>
     <div class="type-specs">
       <p>
         <strong>Body / Small</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:556]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 4]
```diff
@@ -571,7 +571,7 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
       <p>
         <strong>Body / X-Small</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:574]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 5]
```diff
@@ -591,27 +591,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-        This is used for layout headings
+        This is commonly used for H1 headings
       </Paragraph>
     </div>
     <div class="type-specs specs-desktop">
       <p>
         <strong>Heading / X-Large</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:601]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 6]
```diff
@@ -591,27 +591,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-      <p>Weight: Semibold / 600</p>
+      <p>Weight: Bold / 700</p>
       <p>Letter-spacing: 0px</p>
     </div>
     <div class="type-specs specs-mobile">
       <p>
         <strong>Heading / X-Large</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:611]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 7]
```diff
@@ -622,27 +622,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-        This is used for layout headings
+        This is commonly used for H2 headings
       </Paragraph>
     </div>
     <div class="type-specs specs-desktop">
       <p>
         <strong>Heading / Large</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:632]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 8]
```diff
@@ -622,27 +622,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-      <p>Weight: Semibold / 600</p>
+      <p>Weight: Bold / 700</p>
       <p>Letter-spacing: 0px</p>
     </div>
     <div class="type-specs specs-mobile">
       <p>
         <strong>Heading / Large</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:642]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 9]
```diff
@@ -653,27 +653,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-        This is used for layout headings and component titles
+        This is commonly used for H3 headings
       </Paragraph>
     </div>
     <div class="type-specs specs-desktop">
       <p>
         <strong>Heading / Medium</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:663]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 10]
```diff
@@ -653,27 +653,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-      <p>Weight: Semibold / 600</p>
+      <p>Weight: Bold / 700</p>
       <p>Letter-spacing: 0px</p>
     </div>
     <div class="type-specs specs-mobile">
       <p>
         <strong>Heading / Medium</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:673]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 11]
```diff
@@ -684,27 +684,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-        This is used for layout headings and component titles
+        This is commonly used for H4 headings
       </Paragraph>
     </div>
     <div class="type-specs specs-desktop">
       <p>
         <strong>Heading / Small</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:694]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 12]
```diff
@@ -684,27 +684,27 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-      <p>Weight: Semibold / 600</p>
+      <p>Weight: Bold / 700</p>
       <p>Letter-spacing: 0px</p>
     </div>
     <div class="type-specs specs-mobile">
       <p>
         <strong>Heading / Small</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:704]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 13]
```diff
@@ -715,17 +715,17 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-        This is used for layout headings and component titles
+        This is commonly used for H5 headings
       </Paragraph>
     </div>
     <div class="type-specs">
       <p>
         <strong>Heading / X-Small</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:725]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx [hunk 14]
```diff
@@ -736,14 +736,14 @@ Narrative type uses Noto Serif but is otherwise styled the same as the default P
...
-        This is used for layout headings and component titles
+        This is commonly used for H6 headings
       </Paragraph>
     </div>
     <div class="type-specs">
       <p>
         <strong>Heading / 2X-Small</strong>
       </p>
-      <p>Typface: Open Sans</p>
+      <p>Typface: Noto Serif</p>
```
Comment (aaronmaturen) [inline packages/react-magma-docs/src/pages/design/typography.mdx:746]: typeface?

File: packages/react-magma-docs/src/pages/design/typography.mdx (hunk 1 above)
Comment (orion-cengage) [reply packages/react-magma-docs/src/pages/design/typography.mdx:518]: Great catch!  Fixing now.

**Pull request merged**